pkg/ccl/backupccl/restore_job.go

```diff
@@ -812,18 +812,31 @@ func shouldPreRestore(table *tabledesc.Mutable) bool {
 
 // backedUpDescriptorWithInProgressImportInto returns true if the backed up descriptor represents a table with an in
 // progress import that started in a cluster finalized to version 22.2.
-func backedUpDescriptorWithInProgressImportInto(
-	ctx context.Context, p sql.JobExecContext, desc catalog.Descriptor,
-) (bool, error) {
+func backedUpDescriptorWithInProgressImportInto(desc catalog.Descriptor) bool {
 	table, ok := desc.(catalog.TableDescriptor)
 	if !ok {
-		return false, nil
+		return false
 	}
 
-	if table.GetInProgressImportStartTime() == 0 {
-		return false, nil
+	return table.GetInProgressImportStartTime() > 0
+}
+
+func backedUpDescriptorWithImportEpoch(desc catalog.Descriptor) bool {
```

Comment:
nit: could add a comment here? if a desc an importEpoch, it just means that an import was run at some point on table.